consensus/ConsensusFileGenerator.py

```diff
@@ -153,7 +158,9 @@ def _get_matching_history(self, variant):
         chromosome = variant["chromosome"]
         gene = variant["gene"]
         variant_type = variant['type']
-
+        incorrect_history_file = None
+        if self.incorrect_variant_history_file_name:
+            incorrect_history_file = open(self.incorrect_variant_history_file_name, 'a')
```

Comment:
what happens if you already have an existing file (e.g. you execute this piece of code twice)? will you end up with a different file that is twice as large?

Reply:
See comment above.